src/App.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import style from './App.module.scss';
+import Header from './components/Header/Header';
+import CardItems from './components/CardItems/CardItems';
+import { URL } from './models/enums';
+import { ShowData } from './models/interfaces';
+import { MdReportGmailerrorred } from 'react-icons/md';
+import ErrorBoundary from './components/ErrorBoundary/ErrorBoundary';
+
+interface AppProps {
```

Comment:
Where do you pass props to App component?

Reply:
In line 21, without this interface App class component with TS is showing the error for the constructor